File: frontend-lsport/src/app/entities/FootballElement.ts
export class FootballElement {
  id: number;
  nome: string;
  equipa: string;
  descricao: string;
  photoPath: string;

  constructor(id: number, nome: string, equipa: string, descricao: string, photoPath: string) {
    this.id = id;
    this.nome = nome;
    this.equipa = equipa;
    this.descricao = descricao;
    this.photoPath = photoPath;
  }
}

File: backend-lfsport/src/main/java/com/ngola/backendlfsport/entity/FootballElement.java
package com.ngola.backendlfsport.entity;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
public class FootballElement {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    protected long id;
    protected String nome;
    protected String equipa;
    @Lob
    @Column(columnDefinition = "LONGTEXT")
    protected String descricao;
    protected String photoPath;
}
